embulk-util-retryhelper-jetty92/src/main/java/org/embulk/util/retryhelper/jetty92/Jetty92RetryHelper.java

```diff
@@ -177,6 +177,9 @@ public void onGiveup(Exception first, Exception last)
         }
         catch (RetryGiveupException ex) {
             // RetryGiveupException is ExecutionException, which must not be RuntimeException.
```

Comment:
This comment seems to be not much relevant? Since what we deal with is the cause of RetryGiveupException, which is not known whether it's a RuntimeException or not. Let me know if you wanted to remove it.

Reply:
Ah... got it. It's a mistake in https://github.com/embulk/embulk-util-retryhelper/pull/3/files#diff-fb2c90eee63b8309d90322d0754e2b8f when I removed Guava's `Throwables`.

You're right. This comment should be removed. Thanks for the good catch!